### DISCUSSION TITLE
help with live streaming commands using RTMP

### DISCUSSION THREAD

**Discussion by rcleaves** [gopro/OpenGoPro#385] [Q&A]
I am writing an Android app and have gone through the tutorials but can't find the code/commands to start an RMTP live stream. This can be done using the Quick app. How can I start a live stream over RTMP using the Open GoPro SDK? From the Open GoPro documentation: "The API provides 3 different ways to stream videos". Live Stream is one of them, however, the API documentation does not contain the commands to do this and the sample tutorial code and demos do not include this as well. 

![image](https://github.com/gopro/OpenGoPro/assets/887282/8bf359f3-cd95-4c89-9b0f-f081131676c1)

Here are the HTTP commands that are documented (no live stream support)

![image](https://github.com/gopro/OpenGoPro/assets/887282/a18ece68-c970-4d4f-bd13-9cbcc453f7af)

**Comment by hemantbhr** [comment]
@rcleaves I can help with Live Streaming over https with gopro sdk .. please connect with me on whatsapp +919907099989 or skype me on hemant.bohra1987
